src/dftd4/param.f90

```diff
@@ -45,18 +45,196 @@ module dftd4_param
+   call funcs(p_lcwpbe)%set_names([character(len=20) :: 'lc-wpbe', 'lcwpbe'])
+   call funcs(p_bmk)%set_names([character(len=20) :: 'bmk'])
+   call funcs(p_b1b95)%set_names([character(len=20) :: 'b1b95'])
+   call funcs(p_pwb6k)%set_names([character(len=20) :: 'bwb6k'])
+   call funcs(p_otpss)%set_names([character(len=20) :: 'otpss'])
+   call funcs(p_ssb)%set_names([character(len=20) :: 'ssb'])
+   call funcs(p_revssb)%set_names([character(len=20) :: 'revssb'])
+   call funcs(p_pbesol)%set_names([character(len=20) :: 'pbesol'])
+   call funcs(p_pbexalpha)%set_names([character(len=20) :: 'pbexalpha'])
+   call funcs(p_pbehpbe)%set_names([character(len=20) :: 'pbehpbe'])
+   call funcs(p_hcth407)%set_names([character(len=20) :: 'hcth407'])
+   call funcs(p_n12)%set_names([character(len=20) :: 'n12'])
+   call funcs(p_pkzb)%set_names([character(len=20) :: 'pkzb'])
+   call funcs(p_thcth)%set_names([character(len=20) :: 'thcth', 'tauhctc'])
+   call funcs(p_m11l)%set_names([character(len=20) :: 'm11l'])
+   call funcs(p_mn15l)%set_names([character(len=20) :: 'mn15l'])
+   call funcs(p_mpwb1k)%set_names([character(len=20) :: 'mpwb1k'])
+   call funcs(p_mpw1kcis)%set_names([character(len=20) :: 'mpw1kcis'])
+   call funcs(p_mpwkcis1k)%set_names([character(len=20) :: 'mpwkcis1k'])
+   call funcs(p_mpw1b95)%set_names([character(len=20) :: 'mpw1b95'])
+   call funcs(p_pbeh1pbe)%set_names([character(len=20) :: 'pbeh1pbe'])
+   call funcs(p_pbe1kcis)%set_names([character(len=20) :: 'pbe1kcis'])
+   call funcs(p_b97_1)%set_names([character(len=20) :: 'b97-1'])
+   call funcs(p_b97_2)%set_names([character(len=20) :: 'b97-2'])
+   call funcs(p_b98)%set_names([character(len=20) :: 'b98'])
+   call funcs(p_hiss)%set_names([character(len=20) :: 'hiss'])
+   call funcs(p_hse03)%set_names([character(len=20) :: 'hse03'])
+   call funcs(p_hse06)%set_names([character(len=20) :: 'hse06'])
+   call funcs(p_hse12)%set_names([character(len=20) :: 'hse12'])
+   call funcs(p_hse12s)%set_names([character(len=20) :: 'hse12s'])
+   call funcs(p_hsesol)%set_names([character(len=20) :: 'hsesol'])
+   call funcs(p_revtpssh)%set_names([character(len=20) :: 'revtpssh'])
+   call funcs(p_tpss1kcis)%set_names([character(len=20) :: 'tpss1kcis'])
+   call funcs(p_m05)%set_names([character(len=20) :: 'm05'])
+   call funcs(p_m052x)%set_names([character(len=20) :: 'm052x', 'm05-2x'])
+   call funcs(p_m08hx)%set_names([character(len=20) :: 'm08hx', 'm08-hx'])
+   call funcs(p_lcwhpbe)%set_names([character(len=20) :: 'lcwhpbe', 'lc-whpbe'])
+   call funcs(p_mn12l)%set_names([character(len=20) :: 'mn12l'])
+   call funcs(p_tauhcthhyb)%set_names([character(len=20) :: 'tauhcthhyb'])
+   call funcs(p_sogga11x)%set_names([character(len=20) :: 'sogga11x'])
+   call funcs(p_n12sx)%set_names([character(len=20) :: 'n12sx'])
+   call funcs(p_mn12sx)%set_names([character(len=20) :: 'mn12sx'])
+   call funcs(p_mn15)%set_names([character(len=20) :: 'mn15'])
```

Comment:
MN15-D4 e.g. seems to be supported
https://github.com/dftd4/dftd4/blob/3844dc141936e801ca059d1daea1773c3e6103ec/src/dftd4/param.f90#L819-L820
but the parameters are missing.